pyHS100/tests/test_pyHS100.py

```diff
@@ -261,15 +262,69 @@ def test_mac(self):
         # TODO check setting?
 
 
-class TestSmartPlugHS100(TestSmartPlugHS110):
-    SYSINFO = sysinfo_hs100
+class TestSmartPlugHS110(TestSmartPlugHS100):
+    SYSINFO = sysinfo_hs110
+
+    def test_emeter_upcast(self):
```

Comment:
Perhaps this should be a separate testcase class for `EmeterStatus` class?

Reply:
Good point, I'll fix that.